std/net/curl.d

```diff
@@ -313,6 +313,7 @@ version(unittest)
 }
 version(StdDdoc) import std.stdio;
 
+///
 extern (C) void exit(int);
```

Comment:
Ehm? It just "imports" libc's exit function AFAICT, no need to make it public, rather it should be private